OpenRA.Mods.Common/Traits/Guard.cs

```diff
@@ -50,6 +50,9 @@ public void GuardTarget(Actor self, Target target, bool queued = false)
 			if (target.Type != TargetType.Actor)
 				return;
 
+			if (!self.AcceptsOrder("Guard"))
```

Comment:
This will make the unit ignore scripted guard activities as well as player issued orders. This should be placed inside the `ResolveOrder` method instead

Reply:
That is good to know. Fixed.